jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>I thought the wording here was awkward but it turns out it's just exactly copied from https://www.sciencedirect.com/topics/immunology-and-microbiology/beta-interferon -_-</p><p> </p><p>I actually think you should delete the entire plagiarized section anyway because it doesn't contribute any understanding.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>